util/instance.cc

```diff
@@ -398,9 +398,9 @@ bool Instance::run()
                 case SSL_ERROR_SSL:
                 default:
                   {
-                    char cyassl_error_buffer[SSL_ERROR_SIZE]= { 0 };
-                    ERR_error_string_n(ssl_error, cyassl_error_buffer, sizeof(cyassl_error_buffer));
-                    _last_error= cyassl_error_buffer;
+                    char wolfssl_error_buffer[SSL_ERROR_SIZE]= { 0 };
+                    ERR_error_string_n(ssl_error, wolfssl_error_buffer, sizeof(wolfssl_error_buffer));
+                    _last_error= wolfssl_error_buffer;
```

Comment:
I think the `cyassl_*`/`wolfssl_*` variable names here and on the previous two lines should be `ssl_*` since this code appears to be used with both WolfSSL and OpenSSL.